mypy/checker.py

```diff
@@ -2351,7 +2351,14 @@ def check_override(
         # Use boolean variable to clarify code.
         fail = False
         op_method_wider_note = False
-        if not is_subtype(override, original, ignore_pos_arg_names=True):
+        import sys
+
+        if not is_subtype(
+            # HACK: Extremely cringe, but oh well
```

Comment:
whats cringe about it

Reply:
It's checking that the mypy project config file is present in the program arguments. This small change would introduce thousands of errors and make the entire project invalid, and I can't think of a better way to handle it other than introduce a flag for "work properly".

Reply:
arent there already checks like "is_based" or something for stuff like that?

Reply:
or u could just baseline the new errors?